.github/workflows/foreman_plugin.yml

```diff
@@ -171,7 +173,10 @@ jobs:
           POSTGRES_PASSWORD: password
     strategy:
       fail-fast: false
-      matrix: ${{ fromJson(needs.setup_matrix.outputs.matrix) }}
+      matrix:
+        ruby: ${{ fromJson(needs.setup_matrix.outputs.matrix).ruby }}
+        node: ${{ fromJson(needs.setup_matrix.outputs.matrix).node }}
+        postgresql: ${{ fromJson(needs.setup_matrix.outputs.matrix).postgresql }}
```

Comment:
this is needed, as otherwise `exclude: [{'task': 'db:seed'}]` fails with:

Error when evaluating 'strategy' for job 'existing-database'. theforeman/actions/.github/workflows/foreman_plugin.yml@plugin-include-exclude (Line: 176, Col: 15): Matrix exclude key 'task' does not match any key within the matrix


Which makes me wonder how useful these include/excludes are after all…

Reply:
I like the consistency with lines 76 - 79